tests/Tests/Serialise.hs

```diff
@@ -8,16 +8,50 @@ module Tests.Serialise
 #if !MIN_VERSION_base(4,8,0)
 import           Data.Word
 #endif
+
+import           Data.Hashable
 import           Data.Typeable
 
-import           Test.Tasty
 import           Test.QuickCheck
+import           Test.Tasty
 import           Test.Tasty.QuickCheck
 
 import           Data.Binary.Serialise.CBOR
 import           Data.Binary.Serialise.CBOR.Decoding
 import           Data.Binary.Serialise.CBOR.FlatTerm
 
+import qualified Data.HashMap.Strict        as HashMap
+import qualified Data.HashSet               as HashSet
+import qualified Data.IntMap                as IntMap
+import qualified Data.IntSet                as IntSet
+import qualified Data.Map                   as Map
+import qualified Data.Sequence              as Sequence
+import qualified Data.Set                   as Set
+import qualified Data.Foldable              as Foldable
+
+-- TODO use a package for this
```

Comment:
Add whitespace between the instance clauses. And wrap this whole block of instances in:

```
#if !MIN_VERSION_QuickCheck(2,8,2)
...
#endif
```

This will ensure the build passes on Travis, since it uses `cabal-install` (which will pick fresher packages than current Stackage LTS will, notably `QuickCheck-2.8.2`.)